addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2173,6 +2171,16 @@ var SnippetsMenu = Widget.extend({
     execWithLoadingEffect(action, contentLoading = true, delay = 500) {
         return this._execWithLoadingEffect(...arguments);
     },
+    /**
+     * Public method to update the 'savingContent' status. This indicator will be
+     * used to prevent some unwanted actions on save (e.g. a click on the page or
+     * an update on options visibility while destroying the editors...)
+     *
+     * @param {boolean} status
+     */
+    toggleSavingStatus(status) {
+        this.savingContent = status;
+    },
```

Comment:
Actually, following what I explained above, this flag/method could by at the wysiwyg level -> at the end, better have it in the top most widget which receives the "save" action. And which is accessible everywhere via `this.options.wysiwyg`.